packages/react-components/react-tags/src/components/TagGroup/TagGroup.types.ts

```diff
@@ -0,0 +1,18 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { TagSize } from '../Tag/Tag.types';
+
+export type TagGroupSlots = {
+  root: Slot<'div'>;
+};
+
+/**
+ * TagGroup Props
+ */
+export type TagGroupProps = ComponentProps<TagGroupSlots> & {
+  size?: TagSize;
+};
+
+/**
+ * State used in rendering TagGroup
+ */
+export type TagGroupState = ComponentState<TagGroupSlots> & Required<Pick<TagGroupProps, 'size'>>;
```

Comment:
Would be good to have a `TagGroupContextValues` type as similar to other components that present contextual value: https://github.com/microsoft/fluentui/blob/cd60ce9b960b7c107fee94ff41288540e917789a/packages/react-components/react-menu/src/components/Menu/Menu.types.ts#L171

Reply:
Oh I see! yeah let me do this change